src/ResourceManager/Storage/Commands.Management.Storage/packages.config

```diff
@@ -3,4 +3,4 @@
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Storage" version="7.1.0-preview" targetFramework="net452" />
   <package id="WindowsAzure.Storage" version="8.1.1" targetFramework="net452" />
```

Comment:
You will need to update the version here as well

Reply:
fixed